test/absinthe/execution/arguments/scalar_test.exs

```diff
@@ -26,12 +26,12 @@ defmodule Absinthe.Execution.Arguments.ScalarTest do
 
   @graphql """
   query {
-    raisingThing(name: {firstName: "bob"})
+    raisingThing(name: [{firstName: "bob"}])
```

Comment:
this was a valid scalar after this change so switched it to a list that is not a valid scalar.  Or should it be now?

Reply:
Yeah, this is the challenge with this feature, ideally these tests should not need to change. We need to make sure we aren't deviating from the spec here. This may require that the scalars have some sort of explicit flag.

Reply:
Alright added handling for that open_ended flag